src/views/CardStack/CardStack.js

```diff
@@ -272,9 +273,11 @@ class CardStack extends Component {
         // Measure the distance from the touch to the edge of the screen
         const screenEdgeDistance = currentDragPosition - currentDragDistance;
         // Compare to the gesture distance relavant to card or modal
-        const gestureResponseDistance = isVertical
-          ? GESTURE_RESPONSE_DISTANCE_VERTICAL
-          : GESTURE_RESPONSE_DISTANCE_HORIZONTAL;
+        const gestureResponseDistance =
+          this.props.gestureResponseDistance ||
```

Comment:
maybe `gestureResponseDistance` could be an object containing `vertical` and `horizontal` properties so you can individually set response distances for vertical and horizontal?